docs/src/model_creation/examples/basic_CRN_library.md

```diff
@@ -1,4 +1,22 @@
 # [Library of Basic Chemical Reaction Network Models](@id basic_CRN_library)
+```@raw html
+<details><summary><strong>Tutorial environment set-up</strong></summary>
+```
+The following code sets up an environment for running this tutorial.
+```julia
```

Comment:
Could also do stuff like creating a new folder here, but people might not always like to do that. Figured his would be the minimal one.